From: Robert Luberda <robert@debian.org>
Date: Tue, 23 Mar 2010 22:04:47 +0100
Subject: Remove call to execvp()

Merge the following change from Ubuntu:
    - remove call to execvp() as it's not needed and has potential
      security problems.
---
 sar.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/sar.c b/sar.c
index 5d36ebd..44672a5 100644
--- a/sar.c
+++ b/sar.c
@@ -1701,7 +1701,6 @@ int main(int argc, char **argv)
 #ifdef DEBUG
 		fprintf(stderr, "%s: 2.sadc: %s\n", __FUNCTION__, SADC);
 #endif
-		execvp(SADC, args);
 		/*
 		 * Note: Don't use execl/execlp since we don't have a fixed number of
 		 * args to give to sadc.
